mapping.ts

```diff
@@ -6,6 +6,88 @@
  */
 export default {
   icons: {
+    mdiAccount: 'fas-user',
+    mdiAccountCircleOutline: 'fas-user-circle',
+    mdiAccountGroup: 'fas-users',
+    mdiAccountPlus: 'fas-user-plus',
+    mdiAlertCircleOutline: 'fas-exclamation-circle',
+    mdiAlertOutline: 'fas-exclamation-triangle',
+    mdiArrowDown: 'fas-arrow-down',
+    mdiArrowLeft: 'fas-arrow-left',
+    mdiArrowRight: 'fas-arrow-right',
+    mdiArrowUp: 'fas-arrow-up',
+    mdiBackupRestore: 'fas-undo',
+    mdiCalendarRange: 'fas-calendar-alt',
+    mdiChartPie: 'fas-pie-chart',
+    mdiCheck: 'fas-check',
+    mdiCheckBold: 'here is no equivalent in fontawesome',
```

Comment:
It's the tick icon that appears after successfully sharing a file. In our screen designs it looks like the "regular" tick icon. Let's map to that instead.